src/arrayMethodSort.js

```diff
@@ -5,7 +5,41 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    function compareStr(a, b) {
+      const strA = String(a);
+      const strB = String(b);
+
+      if (strA > strB) {
+        return 1;
+      }
+
+      if (strA < strB) {
+        return -1;
+      }
+
+      return 0;
+    }
```

Comment:
```suggestion
    function compareStr(a, b) {
      const strA = String(a);
      const strB = String(b);

      if (stringA === stringB) {
    return 0;
  }

  return stringA > stringB ? 1 : -1;
  
    }
```
let's move the `compareStr` function out from the `applyCustomSort`